velox/exec/tests/TableWriteTest.cpp

```diff
@@ -93,6 +94,31 @@ class TableWriteTest : public HiveConnectorTestBase {
     return getRecursiveFiles(directoryPath).size();
   }
 
+  // Returns a table insert plan node.
+  // Helper method used for "immutablePartitions" set of tests.
+  PlanNodePtr createInsertPlan(
+      const std::vector<RowVectorPtr>& input,
+      const RowTypePtr& outputRowType,
+      const connector::hive::LocationHandle::TableType& outputTableType,
+      const std::string& outputDirectoryPath) {
+    return PlanBuilder()
```

Comment:
Are there other places where this code is duplicated? For example, TEST_F(TableWriteTest, constantVectors) ?

Let's consider all places where we create a plan and figure out what's the best way to avoid copy-paste.

Reply:
I have updated other tests where this is duplicated. However, I left two tests unchanged because they test different output columns and column reordering so it would complicate the helper function.